testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -1274,6 +1274,8 @@ public void verifyFT85() throws IOException, InterruptedException {
     /**
      * To verify the that Save Inbox call Details API request should not succeed with content provided for only 1
      * subscription pack and not even a place holder for second Pack details
+     *
+     * //TODO : FT case to be modified .Refer NMS -182
```

Comment:
Code is modified, only thing left is Ft doc

Thanks,
Abhishek
On Jun 25, 2015 8:41 AM, "Frank Huster" <notifications@github.com> wrote:

> In
> testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java
> <https://github.com/motech-implementations/mim/pull/425#discussion_r33269436>
> :
>
> > @@ -1274,6 +1274,8 @@ public void verifyFT85() throws IOException, InterruptedException {
> >      /**
> >       * To verify the that Save Inbox call Details API request should not succeed with content provided for only 1
> >       * subscription pack and not even a place holder for second Pack details
> > +     *
> > +     * //TODO : FT case to be modified .Refer NMS -182
>
> Why not fix this as part of this pull request?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/425/files#r33269436>.
>